website/css/main-compiled.css

```diff
@@ -11,6 +11,18 @@
   background-color: #AACACC;
```

Comment:
Can you gitignore this file? No point committing compiled code.

Reply:
Yep, just added it